packages/mobile/src/screens/chat-screen/ChatUserListItem.tsx

```diff
@@ -147,10 +147,13 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   }
 }))
 
-const ctaToTextMap = {
+const ctaToTextMap: Record<ChatPermissionAction, string> = {
   [ChatPermissionAction.TIP]: messages.ctaTip,
   [ChatPermissionAction.UNBLOCK]: messages.ctaBlock,
-  [ChatPermissionAction.NONE]: messages.ctaNone
+  [ChatPermissionAction.NONE]: messages.ctaNone,
+  [ChatPermissionAction.WAIT]: messages.ctaNone,
+  [ChatPermissionAction.NOT_APPLICABLE]: messages.ctaNone,
```

Comment:
this will never show up because we check `canMessage` but just covering all bases here.